modules/aleph/src/Aleph/AlephMaterial.php

```diff
@@ -114,6 +121,145 @@ public function isRenewable() {
   }
 
   /**
+   * Checks if the material is available.
+   *
+   * @return bool
+   *    If the material is available.
+   */
+  public function isAvailable() {
+    return $this->available;
+  }
+
+  /**
+   * Checks if the material is reservable.
+   *
+   * @return bool
+   *    If the material is reservable.
+   */
+  public function isReservable() {
+    return $this->reservable;
+  }
+
+  /**
+   * Set the type of material.
+   *
+   * @param string $type
+   *    The type of material (z30-material).
+   */
+  public function setType($type) {
+    $this->type = $type;
+  }
+
+  /**
+   * Get the type of material.
+   *
+   * @return string
```

Comment:
Consider adding examples from real data in the description of all the non-trivial getters. It is a bit of work, but very much worth it for the next person who looks at using the getter.